### PR TITLE
Compatible with parsing for block_cache_usage

### PR DESCRIPTION
**Pull request by jjz921024**
Update for https://github.com/apache/kvrocks/pull/2242
Whether the two fields (`block_cache_pinned_usage`, `index_and_filter_cache_usage`) need to be modified accordingly?

### CODE REVIEW

File: exporter/info.go [hunk 1]
```diff
@@ -249,13 +249,24 @@ func (e *Exporter) handleMetricsReplication(ch chan<- prometheus.Metric, masterH
...
+			}
+			// return ASAP
+			return
+		}
+	}
+
 	prefixs := []string{
-		"block_cache_usage", "block_cache_pinned_usage", "index_and_filter_cache_usage", "estimate_keys",
-		"level0_file_limit_slowdown", "level0_file_limit_stop", "pending_compaction_bytes_slowdown",
+		"block_cache_pinned_usage", "index_and_filter_cache_usage", "estimate_keys", "level0_file_limit_slowdown", "level0_file_limit_stop", "pending_compaction_bytes_slowdown",
```
Comment (git-hulk) [inline exporter/info.go:265]: @jjz921024 Guess we can keep `block_cache_usage` here to be compatible with the old version of Kvrocks server.

**Comment by git-hulk** [comment]
@jjz921024 Thanks for taking care of this.

File: exporter/info.go (hunk 1 above)
Comment (jjz921024) [reply exporter/info.go]: Yes, update it, please review

**Review by git-hulk — APPROVED** [review]

**Comment by git-hulk** [comment]
@jjz921024 Thanks for taking care of this.

**Pull request merged**